Python/di_sensors/inertial_measurement_unit.py

```diff
@@ -13,60 +13,107 @@
         return self.BNO055.read_magnetometer()
-    
+
     def read_gyroscope(self):
-        """Read the gyroscope
-        
-        Returns the current gyroscope (angular velocity) reading as a tuple of X, Y, Z values in degrees per second."""
+        """
+        Read the angular velocity of the gyroscope.
+
+        :returns: The angular velocity as a tuple of X, Y, Z values in *degrees/s*. You can check the X, Y, Z axes on the sensor itself.
+        :rtype: (float,float,float)
+        :raises ~exceptions.OSError: When the sensor is not reachable.
+
+        """
         return self.BNO055.read_gyroscope()
-    
+
     def read_accelerometer(self):
-        """Read the accelerometer
-        
-        Returns the current accelerometer reading as a tuple of X, Y, Z values in meters/second^2."""
+        """
+        Read the accelerometer.
+
+        :returns: A tuple of X, Y, Z values in *meters/(second^2)* units. You can check the X, Y, Z axes on the sensor itself.
+        :rtype: (float,float,float)
+        :raises ~exceptions.OSError: When the sensor is not reachable.
+
+        """
         return self.BNO055.read_accelerometer()
-    
+
     def read_linear_acceleration(self):
-        """Read linear acceleration
-        
-        Returns the current linear acceleration (acceleration from movement not from gravity) reading as a tuple of X, Y, Z values in meters/second^2."""
+        """
+        Read the linear acceleration - that is, the acceleration from movement and without the gravitational acceleration in it.
+
+        :returns: The linear acceleration as a tuple of X, Y, Z values measured in *meters/(second^2)* units. You can check the X, Y, Z axes on the sensor itself.
+        :rtype: (float,float,float)
+        :raises ~exceptions.OSError: When the sensor is not reachable.
+
+        """
         return self.BNO055.read_linear_acceleration()
-    
+
     def read_gravity(self):
-        """Read gravity
-        
-        Returns the current gravity reading as a tuple of X, Y, Z values in meters/second^2."""
+        """
+        Read the gravitational acceleration.
+
+        :returns: The gravitational acceleration as a tuple of X, Y, Z values in *meters/(second^2)* units. You can check the X, Y, Z axes on the sensor itself.
+        :rtype: (float,float,float)
+        :raises ~exceptions.OSError: When the sensor is not reachable.
+
+        """
         return self.BNO055.read_gravity()
-    
+
     def read_quaternion(self):
-        """Read the quaternion values
-        
-        Returns the current orientation as a tuple of X, Y, Z, W quaternion values."""
+        """
+        Read the quaternion values.
+
+        :returns: The current orientation as a tuple of X, Y, Z, W quaternion values.
+        :rtype: (float,float,float,float)
+        :raises ~exceptions.OSError: When the sensor is not present.
+
+        """
         return self.BNO055.read_quaternion()
-    
+
     def read_temperature(self):
```

Comment:
@mattallen37 this method here doesn't report the correct temperature. On my setup it returned values around 1000, so I think something's wrong.